src/scripts/clipperUI/panels/successPanel.tsx

```diff
@@ -9,12 +9,27 @@ import {ClipMode} from "../clipMode";
 import {ClipperStateProp} from "../clipperState";
 import {ComponentBase} from "../componentBase";
 import {Clipper} from "../frontEndGlobals";
+import {RatingsHelper, RatingsPromptStage} from "../ratingsHelper";
 import {Status} from "../status";
 
 import {SpriteAnimation} from "../components/spriteAnimation";
 
-class SuccessPanelClass extends ComponentBase<{ }, ClipperStateProp> {
-	onLaunchOneNoteButton() {
+import {DialogButton, DialogPanel} from "./dialogPanel";
+
+interface SuccessPanelState {
+	currentRatingsPromptStage?: RatingsPromptStage;
+	userSelectedRatingsPromptStage?: RatingsPromptStage;
+}
+
+export class SuccessPanelClass extends ComponentBase<SuccessPanelState, ClipperStateProp> {
+	getInitialState(): SuccessPanelState {
+		return {
+			currentRatingsPromptStage: RatingsPromptStage.INIT,
```

Comment:
You've said this once before on another PR, but to be honest I don't know what you mean by it. Want to open a bug/issue describing this task? :)

#Pending